library/systemd/test/test_helper.rb

```diff
@@ -6,6 +6,21 @@
 Yast.import "SystemdService"
 Yast.import "SystemdTarget"
 
+# Find a Term in given content
+#
+# @param content [Yast::Term] the content in which perform the search
+# @param type [Symbol] wanted Term type
+# @param id [Symbol] wanted Term id
+#
+# @return [Yast::Term|nil] a Term when found; nil otherwise
```

Comment:
NP: I am not sure if `|` delimiter is accepted in YARD, but we usually use comma: `[Yast::Term, nil] `